src/api/PDFPage.ts

```diff
@@ -1441,8 +1427,9 @@ export default class PDFPage {
    * @param options The options to be used when drawing the ellipse.
    */
   drawCircle(options: PDFPageDrawCircleOptions = {}): void {
-    const { size } = options;
+    let { size } = options;
     assertOrUndefined(size, 'size', ['number']);
+    size = asNumber(size ?? 100);
```

Comment:
If we want to set the default value for `size` explicitly there's a slightly cleaner way to do that:
```suggestion
    const { size = 100 } = options;
    assertOrUndefined(size, 'size', ['number']);
```

Reply:
Great, I didn't even know about this! :)